backend/Dockerfile

```diff
@@ -1,26 +1,29 @@
 FROM ubuntu:20.04 as build
 
-RUN apt-get update \
-    && apt-get install --assume-yes --no-install-recommends curl python3-pip \
+RUN apt-get update --quiet --quiet \
+    && apt-get install --quiet --quiet --assume-yes --no-install-recommends \
+    curl python3-pip < /dev/null > /dev/null \
     && rm -rf /var/lib/apt/lists/*
+# hadolint ignore=DL3059
```

Comment:
Weird that hadolint didn't complain about this before.

Reply:
I had to upgrade my hadolint version. Apparently this check came with new version.